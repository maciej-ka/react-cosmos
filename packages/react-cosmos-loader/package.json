{
  "name": "react-cosmos-loader",
  "version": "2.0.0-beta.20",
  "description": "Loader for rendering React components in isolation",
  "repository": "https://github.com/react-cosmos/react-cosmos/tree/master/packages/react-cosmos-loader",
  "license": "MIT",
  "files": [
    "lib"
  ],
  "main": "lib/index.js",
  "dependencies": {
    "lodash.merge": "^4.6.0",
    "prop-types": "^15.5.10",
    "react-cosmos-state-proxy": "^2.0.0-beta.20",
    "react-cosmos-utils": "^2.0.0-beta.20"
  },
  "xo": false
}
