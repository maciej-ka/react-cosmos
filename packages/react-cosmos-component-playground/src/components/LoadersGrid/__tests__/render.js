import React from 'react';
import { Loader } from 'react-cosmos-loader';
import renderer from 'react-test-renderer';
import StarryBg from '../';
import blankFixture from '../__fixtures__/blank';

describe('StarryBg blank', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(
      <Loader component={StarryBg} fixture={blankFixture} />
    );
  });

  it('renders correctly', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
