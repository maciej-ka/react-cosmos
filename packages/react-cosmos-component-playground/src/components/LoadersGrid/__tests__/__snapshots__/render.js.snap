// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarryBg blank renders correctly 1`] = `
<div>
  Grid of loaders with one for each fixture
</div>
`;
